implement-Array-prototype-reduce.js

```diff
@@ -0,0 +1,15 @@
+Array.prototype.myReduce = function(func, initialValue) {
+  let result = initialValue ?? this[0];
+  let startIndex = (initialValue === undefined)
+    ? 1
+    : 0;
+
+  for (let i = startIndex; i < this.length; i++) {
+    result = func(result, this[i]);
```

Comment:
super
maybe you forgot about passing of the index and the array itself to the callback function